microsoft-edge/devtools-guide-chromium/evaluate-performance/selector-stats.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
@zoherghadyali - add entry in TOC.yml

https://github.com/MicrosoftDocs/edge-developer/blob/user/zoghadya/selector-stats/microsoft-edge/toc.yml

Reply:
this was done so can Resolve this comment thread